.github/workflows/browser-tests.yml

```diff
@@ -0,0 +1,129 @@
+                default: 'ezsystems/php:7.4-v2-node14'
+                description: 'The PHP image to use'
+                required: false
+                type: string
+            timeout:
+                default: 30
+                description: 'Job maximum timeout in minutes'
+                required: false
+                type: number
+            composer-package-version:
+                default: ''
+                description: 'The version of the package tested'
+                required: false
+                type: string
+        secrets:
+            SATIS_NETWORK_KEY:
+                required: false
+            SATIS_NETWORK_TOKEN:
+                required: false
+            TRAVIS_GITHUB_TOKEN:
+                required: false
+
+env:
+    APP_ENV: behat
+    APP_DEBUG: 1
+    PHP_INI_ENV_memory_limit: 512M
+
+jobs:
+    browser-tests:
+        runs-on: ubuntu-latest
+        timeout-minutes: ${{ inputs.timeout }}
+
+        steps:
+            - if: contains(inputs.setup, 'doc/docker/varnish.yml')
+              name: '[Varnish] Set the URL the tests should access'
+              run: echo "WEB_HOST=http://varnish" >> $GITHUB_ENV
+
+            - uses: actions/checkout@v2
+
+            - name: Setup PHP Action
+              uses: shivammathur/setup-php@v2
+              with:
+                php-version: 7.4
+                coverage: none
+
+            - if: inputs.composer-package-version != ''
+              name: Set COMPOSER_ROOT_VERSION for packages with cyclic dependencies
+              run: echo "COMPOSER_ROOT_VERSION=${{ inputs.composer-package-version }}" >> $GITHUB_ENV
+
+            - if: env.SATIS_NETWORK_KEY != ''
+              name: Add composer keys for private packagist
+              run: |
+                composer config http-basic.updates.ibexa.co $SATIS_NETWORK_KEY $SATIS_NETWORK_TOKEN
+                composer config github-oauth.github.com $TRAVIS_GITHUB_TOKEN
+              env:
+                SATIS_NETWORK_KEY: ${{ secrets.SATIS_NETWORK_KEY }}
+                SATIS_NETWORK_TOKEN: ${{ secrets.SATIS_NETWORK_TOKEN }}
+                TRAVIS_GITHUB_TOKEN: ${{ secrets.TRAVIS_GITHUB_TOKEN }}
+
+            - uses: "ramsey/composer-install@v1"
+              with:
+                dependency-versions: "highest"
+                composer-options: "--prefer-dist --no-progress"
+
+            - name: Set up whole project using the tested dependency
+              run: ./vendor/bin/prepare_project_edition.sh ${{ inputs.project-edition }} ${{ inputs.project-version }} ${{ inputs.setup }} ${{ inputs.php-image }}
+
+            - if: inputs.multirepository
+              name: Set up multirepository build
+              run: $HOME/build/project/vendor/ezsystems/behatbundle/bin/.travis/prepare_multirepository_setup.sh
+
+            - if: inputs.test-setup-phase-1 != ''
+              name: Run first phase of tests setup
+              run: |
+                  cd ${HOME}/build/project
```

Comment:
Don't think `${HOME}/` is needed here, as the working directory is reset each time a new step is instantiated.